### PR TITLE
Templates API v2 documentation

### PR DESCRIPTION
**Pull request by aeden**

### CODE REVIEW

File: content/v2/templates.markdown [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+## Delete a template {#delete}
+
+    DELETE /:account/templates/:template
+
+### Parameters
+
+Name | Type | Description
+-----|------|------------
+`:account` | `integer` | The account id or wildcard (`_`)
+`:template` | `integer` | The template id or short name
```
Comment (weppos) [inline content/v2/templates.markdown]: It should probably be

    `:template` | `integer`, `string` | The template id or short name

File: content/v2/templates.markdown [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+### Input
+
+See [create](#create).
+
+### Response
+
+Responds with HTTP 200 on success.
+
+~~~js
```
Comment (weppos) [inline content/v2/templates.markdown]: I suggest we use

    ~~~json

Comment (aeden) [inline content/v2/templates.markdown]: Odd, I pulled this directly from the contacts documentation (I think).

Comment (weppos) [inline content/v2/templates.markdown]: I've just noticed it recently in a previous commit. It may be possible we'll have to update also some other files.

**Pull request merged**